docs/book/src/reference/commands/alpha_update.md

```diff
@@ -81,17 +87,62 @@ Once updated, you can use `kubebuilder alpha update` for future upgrades.
 | `--from-version`    | **Required for projects initialized with versions earlier than v4.6.0.** Kubebuilder version your project was created with. If unset, uses the `PROJECT` file. |
 | `--to-version`      | Version to upgrade to. Defaults to the latest version.                                                                                                         |
 | `--from-branch`     | Git branch that contains your current project code. Defaults to `main`.                                                                                        |
+| `--force`           | Force the update even if conflicts occur. Conflicted files will include conflict markers, and a commit will be created automatically. Ideal for automation (e.g., cronjobs, CI).                                                                       |
 | `-h, --help`        | Show help for this command.                                                                                                                                    |
-
 <aside class="note warning">
-<h1>Projects generated with </h1>
-
 Projects generated with **Kubebuilder v4.6.0** or later include the `cliVersion` field in the `PROJECT` file.
 This field is used by `kubebuilder alpha update` to determine the correct CLI
 version for upgrading your project.
+</aside>
+
+## Merge Conflicts with `--force`
+
+When you use the `--force` flag with `kubebuilder alpha update`, Git will complete the merge even if there are conflicts. The resulting commit will include conflict markers like:
+```
+<<<<<<< HEAD
+Your changes
+=======
+Incoming changes
+>>>>>>> branch-name
+```
+These conflicts will be committed in the
+`tmp-kb-update-merge` branch.
 
+<aside class="note warning">
+You must manually resolve these conflicts before merging into your main branch.
+
+### Required Steps If Conflicts Are Present
+
+If the merge introduces conflicts, follow these steps before committing any final changes:
+1. Resolve Conflicts;
+2. Open the conflicted files and manually edit them to resolve all merge markers;
+3. Run Required Make Targets;
+4. After resolving, run the following command to regenerate, format, vet, and lint the code:
+```
+make manifests generate fmt vet lint-fix
+```
+
+### Optional (but Recommended)
+
+1. Run the full validation suite to ensure your project remains consistent and working:
+```
+make all
+```
+2. Create Pull Request
+ After validation, commit your changes and push a pull request from `tmp-kb-update-merge` to your target branch.
 </aside>
```

Comment:
````suggestion
```suggestion
<aside class="note warning">
<H1> If you face conflicts </H1>
If the merge introduces conflicts, you must resolve them and **ensure** you execute the following command to regenerate the manifests and organise the files properly:

```bash
make manifests generate fmt vet lint-fix
```

Alternatively, you may want to run:

```bash
make all
```
````
```